pylint_venv.py

```diff
@@ -29,6 +29,10 @@
 
     pylint --init-hook="import pylint_venv; pylint_venv.inithook('$(pwd)/env')"
 
+- If Pylint itself is installed in a different virtualenv, to ignore that virtualenv::
```

Comment:
How about ``If Pylint itself is installed in a virtualenv, to ignore that virtualenv (and activate a different one)::``?

Reply:
That sounds better. Fixing it.